packages/nimble-components/src/table/specs/table-column-specs/table-column-text-field.md

```diff
@@ -0,0 +1,175 @@
+# TableColumnTextField
+
+## Overview
+
+The `nimble-table-column-text-field` is a component that defines how to render a cell for that column in a `nimble-table` as text.
+
+### Background
+
+[Table Column API](../table-columns-hld.md)
+
+[Table Spec](../README.md)
+
+[Visual Design Spec](https://xd.adobe.com/view/5b476816-dad1-4671-b20a-efe796631c72-0e14/specs/)
+
+### Non-goals
+
+-   Defining an API that supports editable text. Once we need editable text we will determine if it makes sense to modify this component or introduce a new column type.
+
+### Features
+
+-   Offers a `placeholder` attribute that allows a user to define what text to render when no value is provided by the `nimble-table` data.
+
+---
+
+## Design
+
+Below is an example of how the `nimble-table-column-text-field` would be used within a `nimble-table`:
```

Comment:
I wonder if calling this `text-field` is a bit misleading since it isn't editable (and as speced isn't even using the `nimble-text-field`). I don't have a great alternative suggestion, but I wonder if calling it something like `text` would be better (i.e. `nimble-table-column-text`).

Reply:
Updated.